client/src/components/Draftail/Draftail.scss

```diff
@@ -206,10 +208,15 @@ $draftail-editor-font-family: $font-sans;
   border-width: 1px;
 }
 
+.Draftail-ComboBox__status {
+  padding-left: 30px;
```

Comment:
Please use `padding-inline-start` and the appropriate theme values.

Reply:
> Please use `padding-inline-start` and the appropriate theme values.

Ok, will work on that

Reply:
> Please use `padding-inline-start` and the appropriate theme values.

Please review again @lb-